lib/Command/PointList.php

```diff
@@ -171,7 +172,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 						($fresh) ? $point->getParent() : '',
 						$comment,
 						implode(',', $status),
-						$instance,
+						($instance === RemoteInstance::LOCAL) ? '<info>' . $instance . '</info>' : $instance,
```

Comment:
Wouldn't it be better to do it in the front-end side ?

Reply:
it's an occ command :]